deploy/general-deploy/kubeturbo.yaml

```diff
@@ -1,30 +1,53 @@
-apiVersion: v1
-kind: Pod
+apiVersion: extensions/v1beta1
```

Comment:
should we put the configMap before the deployment? Otherwise, the pod may not find the configMap when it tries to start.

Reply:
This needs to be addressed in `README`.

Reply:
Yes, `README` will be updated separately as it will also consider other changes to kubeturbo we recently made.